extern/storage-sealing/fsm.go

```diff
@@ -642,3 +746,16 @@ func planOne(ts ...func() (mut mutator, next func(*SectorInfo) (more bool, err e
 		return uint64(len(events)), nil
 	}
 }
+
+// planOne but ignores unhandled states without erroring, this prevents the need to handle all possible events creating
+// error during forced override
+func planOneOrIgnore(ts ...func() (mut mutator, next func(*SectorInfo) (more bool, err error))) func(events []statemachine.Event, state *SectorInfo) (uint64, error) {
```

Comment:
Using on AbortUpgrade right now, could use your input on whether this is sound or not.  I introduced it because forcing state to AbortUpgrade would frequently error because of ongoing events unhandled by the AbortUpgrade state

Reply:
TODO also use for removing